.gitignore

```diff
@@ -1,3 +1,6 @@
 *~
 src/
-__pycache__/
\ No newline at end of file
+__pycache__/
+hello.py
+testing-script.py
+Dockerfile
```

Comment:
I'd have thought a Dockerfile was something useful to commit? Am I missing something?

Reply:
It's templated, so we only need the template, not the result of populating it.